packages/DataHeader/src/DataHeader.styles.js

```diff
@@ -0,0 +1,62 @@
+import styled, { css } from "styled-components";
+import tokens from "@paprika/tokens";
+
+export const Header = styled.div(({ $color = tokens.color.black, $backgroundColor = "transparent", refHeader }) => {
```

Comment:
These defaults for $color and $backgroundColor are kind of hidden, it seems they should be in the `defaultProps` instead. 

But having defaults makes it difficult to override with CSS.  If they can be `null` then we can just do nothing in `resetHeaderParentContainer()`.